modules/permission-management/src/Volo.Abp.PermissionManagement.Application.Contracts/Volo/Abp/PermissionManagement/Integration/PermissionGrantOutput.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Collections.Generic;
+
+namespace Volo.Abp.PermissionManagement.Integration;
+
+public class PermissionGrantOutput
+{
+    public Guid UserId { get; set; }
```

Comment:
Returning the `UserId` is not necessary since the client already knows it.

Reply:
We may check the permission for multiple users, We have to use `UserId`.